solvebio/cli/main.py

```diff
@@ -103,8 +90,8 @@ class SolveArgumentParser(argparse.ArgumentParser):
                             'Options are "append" (default) or "overwrite".'
                 },
                 {
-                    'name': 'dataset_name',
-                    'help': 'The name of the dataset'
+                    'name': 'dataset_full_path',
```

Comment:
Maybe you should use `'flags': 'dataset-full-path'`? How about just `full-path`?

Reply:
if we use flags, then what would the default be? or can you make flags required?